content/en/news/releases/1.25.x/announcing-1.25/change-notes/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Istio 1.25.0 Change Notes
+linktitle: 1.25.0
+subtitle: Major Release
+description: Istio 1.25.0 release notes.
+publishdate: 2024-11-07
```

Comment:
Just a reminder we'll need to edit this when we firm up our release date

Reply:
I have put this as 27-02 now...